display_list/testing/dl_test_snippets.h

```diff
@@ -267,11 +255,6 @@ struct DisplayListInvocation {
   // to |DisplayList.byte_count().
   size_t byte_count() { return sizeof(DisplayList) + byte_count_; }
 
-  int sk_op_count() { return sk_op_count_; }
-  // byte count for the ops with DisplayList overhead as translated
-  // through an SkCanvas interface, comparable to |DisplayList.byte_count().
-  size_t sk_byte_count() { return sizeof(DisplayList) + sk_byte_count_; }
```

Comment:
These 2 fields can be deleted now, but it would cascade into a huge edit of the long list of test cases that are expressed using this structure, so I'm leaving that for a future pass.